lib/launch.js

```diff
@@ -369,6 +351,17 @@ const util = require('util'),
         }
     };
 
+    function makeSession(options, next){
+        log.info("launch#makeSession");
+        options.nReplies = 1; // -n 1
+        if (!options.session) {
+            const printTarget = true;
+                    options.session = new novacom.Session(options.device, printTarget, next);
```

Comment:
Please remove spaces in front of "options.session" keyword.

Reply:
I've update it.